### ISSUE TITLE
To reproduce the results on Cora, PubMed, and Citeseer

### ISSUE THREAD

**Issue by bwdeng20** [LirongWu/MD-GNN#1]
I have tried to  use the (default) hyperparameters under the `param` directory to reproduce the results on `zinc`, `cora`, `pubmed`, `citeseer` datasets. While the hyperparameters for `zinc` work well, all of the rest three groups of hyperparameters fail to catch up with the accuracies reported in the paper.

Could you please show the hyperparameters of MD-GNN on `cora`, `pubmed`, and `citeseer`  to the interested researchers like me.

Thx :)

**Comment by LirongWu** [comment]
Thank you for being interested in our work. Please note that (1) We use DGL to implement all the GNN models on three citation datasets (Cora, Citeseer, and Pubmed). In order to evaluate the model with different splitting strategy (fewer and harder label rates), you need to replace the file (dgl/data/citation_graph.py) with the citation_graph.py provided. (2) The results of four data splitting settings for the Cora, Citeseer and Pubmed datasets are reported in the paper, depending on the parameter "percent", and only the hyperparameters for one of the data splitting settings are provided in the param folder; the hyperparameters for the remaining settings can be searched in a similar way using the hyperparameter search tool NNI. (3) Since different hardware platforms vary greatly, fixed hyperparameters may not give exactly the same results in different hardware, so we suggest you to use NNI to search hyperparameters according to the search space described in the paper. If you still find it difficult after trying, please feel free to contact us and we can provide you with scripts for hyperparameter search and hyperparameters for other data splitting settings, but this may take a little time as we need to organize this work that is almost two years old.

**Comment by bwdeng20** [comment]
Thanks for the swift reply.  

For fast verification, I have only tested three citation datasets with one split setting mentioned in Table 6. Could you please tell me which splits the default hyperparameter files under the `param` directory are for?

**Comment by LirongWu** [comment]
You can refer to the parameter "percent" in the param file, which tells us the percentage of labeled data used for training. For example, when percent=0.005 (default hyperparameter for Cora), it means that there are 0.5% of labeled nodes used for training, which corresponds to the result of TrainPCT=0.5% in Table 6.
